contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -289,6 +311,46 @@ contract VestingRegistryLogic is VestingRegistryStorage {
         return vestings[uid].vestingAddress;
     }
 
+    /**
+     * @dev check if the specific vesting address is team vesting or not
+     * @dev read the vestingType from vestingCreationAndTypes storage
+     *
+     * @param _vestingAddress address of vesting contract
+     *
+     * @return true for teamVesting, false for normal vesting
+     */
+    function isTeamVesting(address _vestingAddress) external view returns (bool) {
+        return (vestingCreationAndTypes[_vestingAddress].isSet &&
+            vestingCreationAndTypes[_vestingAddress].vestingType ==
+            uint256(VestingType.TeamVesting));
+    }
+
+    /**
+     * @dev setter function to register existing vesting contract to vestingCreationAndTypes storage
+     * @dev need to set the function visilibty to public to support VestingCreationAndTypeDetail struct as parameter
```

Comment:
could you pls replace all occurences of `VestingCreationAndTypeDetail` for plural form `VestingCreationAndTypeDetails`

Reply:
PLS IGNORE THIS COMMENT
test comment to check if it does not stuck in Pending state